src/providers/token-provider.ts

```diff
@@ -730,7 +734,11 @@ export class TokenProvider implements ITokenProvider {
           continue;
         }
 
-        const symbol = symbolResult.result[0]!;
+        let symbol = symbolResult.result[0]!;
```

Comment:
nit: probably better to retain 'const' variable, rather than re-assign (e.g. via ternary operator):

```
const symbol = isBytes32 ? parseBytes32String(symbolResult.result[0]!) : symbolResult.result[0]!;
```

Reply:
thanks good suggestion!